earth_enterprise/rpms/build.gradle

```diff
@@ -43,6 +43,17 @@ apply plugin: "nebula.ospackage"
 
 import org.opengee.shell.GeeCommandLine
 
+// Varables used for specifying location of two files
+// 'gee_long_version.txt' and 'gepgcommon'
+def working_dir = "../src/"
+
+// Use the command line option "-PDEBUGRPMS" in order to build
+// for NATIVE-DBG-x86_64.  Default is NATIVE-REL-x86_64
+def native_x86 = ( project.hasProperty( 'DEBUGRPMS' ) ? "NATIVE-DBG-x86_64/" : "NATIVE-REL-x86_64/" )
```

Comment:
Minor: it would be nice to also support the "optimize" option here. In your scons command:
- if you specify `release=1` you get `NATIVE-REL-x86_64`
- if you specify `optimize=1` you get `NATIVE-OPT-x86_64`
- if you specify nothing you get `NATIVE-DBG-x86_64`
It would be nice if the script supported all 3 versions.

Reply:
I can look into modifying SCons to achieve this.

Reply:
I think you should pass the build name to Gradle (`REL`, `OPT`, `DBG`, etc.).  E.g., `project.buildName`.

I propose one of the following ways to easily implement that:

1. Copy the PostGIS build directory to the 'staged install' directory.

1. Create a `.properties` file in the 'staged install' directory, which stores a property value, such as `buildName`.

This way there would be less danger of specifying a `src/NATIVE-${buildName}-x86_64` directory that doesn't correspond to the staged install directory.  The `build.gradle` script already receives a staged install location, so it won't need any new options to support the different builds.

I think we would want to add the name of any non-release build to the package name, or version.

Of course, in the future, it would be best if we don't bundle PostGIS and PostgreSQL with Open GEE.